release/release_gold.sh

```diff
@@ -327,4 +330,37 @@ function test_boms {
 	fi
 }
 
+function update_release_info_date {
+	is_update_release_applicable
+
+	if [ "${?}" == "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}" ]
+	then
+		lc_log INFO "Skipping the release info update."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	lc_cd "${_PROJECTS_DIR}"/liferay-portal-ee
+
+	local product_group_version="$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 1,2)"
+
+	local quarterly_release_branch_name="release-${product_group_version}"
+
+	git branch --delete "${quarterly_release_branch_name}" &> /dev/null
+
+	git fetch --no-tags upstream "${quarterly_release_branch_name}":"${quarterly_release_branch_name}" &> /dev/null
+
+	git checkout "${quarterly_release_branch_name}" &> /dev/null
+
+	local formatted_date=$(date -d @"${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}" +"%B %-d, %Y")
+
+	sed -i -e "s/release.info.date=.*/release.info.date=${formatted_date}/" release.properties
+
+	git add "${_PROJECTS_DIR}/liferay-portal-ee/release.properties"
+
+	git commit -m "Updating the release info date for ${_PRODUCT_VERSION}."
+
+	git push upstream "${quarterly_release_branch_name}"
```

Comment:
@kiwm, in order to create a unit test to the function `update_release_info_date` which doesn't push the changes to the repo, I think you can do the following:
```
if [[ ! " ${@} " =~ " --test " ]]
then
    git add "${_PROJECTS_DIR}/liferay-portal-ee/release.properties"

	git commit -m "Updating the release info date for ${_PRODUCT_VERSION}."

	git push upstream "${quarterly_release_branch_name}"
fi
```
Can you give it a try? :slightly_smiling_face: